README.md

```diff
@@ -2,13 +2,13 @@
 
 Express and simple docker setup for all your PHP development.
 
-### Nginx + PHP
+### Apache + PHP
 
-This branch is still on progress. For more stable, use the nginx branches.
+This branch is still on progress. For more stable, use the apache branches.
```

Comment:
Does this mean I have to remove this line also:


`
### PHP 7 or PHP 5?
You can switch between the latest release of PHP 7 or the latest of PHP 5, by
changing the first line of the file "docker/php/Dockerfile".
```
FROM php:5-fpm`